webhooks-extension/src/components/WebhookBranches/WebhookBranches.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from "react";
```

Comment:
missing copyright header
same on some other new files

Reply:
numerous files including old ones not included in this PR are also missing the copyright header for example `WebhookApp`. To keep this PR small I propose adding the header in another PR

Reply:
I don't include copyright headers in the diff size ;)